package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tachometer",
-  "version": "0.4.7",
+  "version": "0.4.8",
```

Comment:
is it 0.4.8 or 0.4.9?

Reply:
8, sorry, just typed the wrong commit message

Reply:
Wait no, 9. I guess I didn't bump the package.json last time. Oops!